test/quantization/test_numeric_suite_fx.py

```diff
@@ -142,3 +144,96 @@ def compare_and_validate_results(float_model, q_model):
         q_model = convert_fx(prepared_model)
 
         compare_and_validate_results(backup_prepared_model, q_model)
+
+    @skipIfNoFBGEMM
```

Comment:
Lets use @override_qengines instead, this way the test can run even with qnnpack.